src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/RemoveAzureRmDataLakeAnalyticsCatalogItemAclEntry.cs

```diff
@@ -0,0 +1,135 @@
+        public SwitchParameter User { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = true,
+            ParameterSetName = GroupCatalogParameterSetName, HelpMessage = "Remove ACL entry of catalog for group.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = true,
+            ParameterSetName = GroupCatalogItemParameterSetName, HelpMessage = "Remove ACL entry of catalog item for group.")]
+        public SwitchParameter Group { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+            ParameterSetName = UserCatalogParameterSetName, HelpMessage = "The identity of the user to remove.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+            ParameterSetName = GroupCatalogParameterSetName, HelpMessage = "The identity of the group to remove.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+            ParameterSetName = UserCatalogItemParameterSetName, HelpMessage = "The identity of the user to remove.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+            ParameterSetName = GroupCatalogItemParameterSetName, HelpMessage = "The identity of the group to remove.")]
+        [Alias("Id", "UserId")]
+        public Guid ObjectId { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = true,
+            ParameterSetName = UserCatalogItemParameterSetName, HelpMessage = "The type of the catalog item(s).")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = true,
+            ParameterSetName = GroupCatalogItemParameterSetName, HelpMessage = "The type of the catalog item(s).")]
+        [PSArgumentCompleter("Database")]
+        public string ItemType { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = true,
+            ParameterSetName = UserCatalogItemParameterSetName,
+            HelpMessage = "The catalog item path to search within, in the format:" +
+                          "'DatabaseName.<optionalSecondPart>.<optionalThirdPart>.<optionalFourthPart>'.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = true,
+            ParameterSetName = GroupCatalogItemParameterSetName,
+            HelpMessage = "The catalog item path to search within, in the format:" +
+                          "'DatabaseName.<optionalSecondPart>.<optionalThirdPart>.<optionalFourthPart>'.")]
+        [ValidateNotNullOrEmpty]
+        public CatalogPathInstance Path { get; set; }
+
+        [Parameter(Mandatory = false,
+            HelpMessage = "Forces the command to run without asking for user confirmation.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
@idear1203 is the `-Force` parameter necessary? We suggest using the `-Force` parameter when the cmdlet performs additional prompting that the user should be able to skip (_e.g._, deleting an empty folder requires no prompting, but deleting a folder with files in it will prompt the user asking if they want to delete the files too). If this cmdlet doesn't do any additional prompting, then you should remove the `-Force` parameter.

Reply:
Thanks! `-Force` will be removed